pylint/extensions/comparison_placement.py

```diff
@@ -0,0 +1,64 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+"""Checks the placement of comparisons"""
+
+
+from astroid import nodes
+
+from pylint.checkers import BaseChecker, utils
+from pylint.interfaces import IAstroidChecker
+
+REVERSED_COMPS = {"<": ">", "<=": ">=", ">": "<", ">=": "<="}
+COMPARISON_OPERATORS = frozenset(("==", "!=", "<", ">", "<=", ">="))
+
+
+class ComparisonPlacementChecker(BaseChecker):
+    """Checks the placement of constants in comparisons"""
```

Comment:
It feels like it's duplicated from the module docstring, we might as well remove one of them.

Reply:
We would miss out on IntelliSense on the class then. I also saw that we are waiting to add `extensions.docparams` in our `pylintrc` which would requires us to re-add this docstring in the future.

Reply:
Fair enough.